### PR TITLE
Improve naming on all order wizard steps

### PR DESCRIPTION
**Pull request by mkasztelnik**
Naming is improved according to #1216. What is more bug with going back when the step is not visible was fixed.

### CODE REVIEW

**Comment by MinLidia** [comment]
@mkasztelnik Marek , I have strange view when I am trying to change the offer type twice. 
Tried to change Compute offer firstly from Ordarable to External and all was working fine. But then I changed the same offer to Internal ordering. 

Firstly it continue showing steps for External ordering and didn't change to Internal ordering plus  now  button names and even Step names are different. 

So can't test your button names as each time it shows me different view. I assume there might be some problem with previous PR with Offer type based on offer selection. 

First try : 
Screenshot 2019-11-14 at 17 10 39

Second try : 
Screenshot 2019-11-14 at 16 32 20

**Comment by mkasztelnik** [comment]
@MinLidia I have troubles to recreate the issue you described. Can you post step by step instruction how to repeat it?

**Comment by mkasztelnik** [comment]
@MinLidia I made some small improvements. Please try to recreate your issue once again :)

**Comment by MinLidia** [comment]
By clicking button : Cancel order and quit system LOG me in as an user.  Please check 

Screenshot 2019-11-15 at 14 15 50

**Comment by MinLidia** [comment]
Due to changes in the button naming , please change the content on the Access instructions for Open access and External ordering ( 1 step) and please link FAQ page to it.  

Screenshot 2019-11-15 at 14 21 06

CONTENT for External ordering : 

"The Service name - offer name is accessible by external ordering. You can either use the service by clicking on the button 'Order Externally', where EOSC Portal will redirect you to the website and you can order the service or what is highly recommended to add the service to your 'Project' by going to Next steps. It helps you to organise your services and orders into logical blocks related to your use case, gain EOSC experts support and easily access the selected service.
To find out more about Projects in EOSC Marketplace, please refer to our FAQ.  ( please link FAQ page here) 
Screenshot 2019-11-15 at 14 11 01

CONTENT for Open access : 

"The Service name - offer name" is open to everyone, any user can reach the service. You can either directly use the service by clicking on the button 'Go to the service' , where EOSC Portal will redirect you to the service website or what is highly recommended to add the service to your Project by going to Next steps. Project option helps you to organise your services and orders into logical blocks related to your use case, gain EOSC experts support and easily access the selected service.
To find out more about Projects in EOSC Marketplace, please refer to our FAQ. ( please link FAQ page here) 
Screenshot 2019-11-15 at 14 13 23

File: app/controllers/services/application_controller.rb [hunk 1]
```diff
@@ -51,15 +51,67 @@ def next_step_key
...
     def step_key
       raise "Should be implemented in descendent class"
     end
+
+    def step_title(step_name = step_key)
+      I18n.t("service.#{step_name}.title")
+    end
+
+    def next_title
+      if next_visible_step_key == "summary"
```
Comment (martaswiatkowska) [inline app/controllers/services/application_controller.rb]: Maybe ProjectItem::Wizard::STEPS.last or wizard.step_names.last?

**Comment by MinLidia** [comment]
Can you please make the word "Project" BOLD for Open Access.

Screenshot 2019-11-15 at 15 36 28

**Comment by MinLidia** [comment]
If possible, link the FAQ page here. If not, please delete this phase and leave only FAQ. 

Screenshot 2019-11-17 at 18 00 40

**Comment by MinLidia** [comment]
Go to the service link stopped working

Screenshot 2019-11-17 at 18 04 19

**Pull request merged**